index.html

```diff
@@ -64,6 +64,23 @@ <h1>Connect</h1>
         <li><p><img src="EmailPic5.jpg" alt="Email 5 Pic" width="20" height="20"> tiffanycooper0425@gmail.com</p></li>
     </ul> 
     </section>
+    <section>
+		<h2>Leave a Message</h2>
+		<form id="form" name="leave_message" action="" method="get">
+		<br><label for="name"> Enter Name: </label>
```

Comment:
```suggestion
	<h2>Leave a Message</h2>
	<form id="form" name="leave_message" action="" method="get">
		<br>
		<label for="name"> Enter Name: </label>
```
etc.

Consistent indentation makes it easier to read.